airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -166,19 +174,26 @@
 
   @BeforeAll
   public static void init() {
-    sourcePsql = new PostgreSQLContainer("postgres:13-alpine")
-        .withUsername(SOURCE_USERNAME)
-        .withPassword(SOURCE_PASSWORD);
-    sourcePsql.start();
+    if (!IS_GKE) {
+      sourcePsql = new PostgreSQLContainer("postgres:13-alpine")
+          .withUsername(SOURCE_USERNAME)
+          .withPassword(SOURCE_PASSWORD);
+      sourcePsql.start();
+    }
   }
 
   @AfterAll
   public static void end() {
-    sourcePsql.stop();
+    if (IS_GKE && !IS_KUBE) {
+      throw new RuntimeException("KUBE Flag should also be enabled if GKE flag is enabled");
```

Comment:
should this check be in the @BeforeAll method?

Reply:
Ah yes!